lib/vfscore/volumes.c

```diff
@@ -0,0 +1,59 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+/* Copyright (c) 2023, Unikraft GmbH and The Unikraft Authors.
+ * Licensed under the BSD-3-Clause License (the "License").
+ * You may not use this file except in compliance with the License.
+ */
+
+#include <string.h>
+#include <sys/mount.h>
+#include <uk/init.h>
+#include <uk/libparam.h>
+
+#define UK_MAX_VFSCORE_VOLUMES_COUNT			64
+#define UK_VFSCORE_VOLUMES_ARGS_SEP			":"
+
+static char *vfscore_volumes[UK_MAX_VFSCORE_VOLUMES_COUNT];
+
+UK_LIBPARAM_PARAM_ARR_ALIAS(automount, &vfscore_volumes, charp,
+			    UK_MAX_VFSCORE_VOLUMES_COUNT, "VFSCore Volumes");
+
+/**
+ * Expected command-line argument format:
+ *	vfs.automount=[
+ *		"<source_device>:<mount_target>:<fs_driver>:<options>"
```

Comment:
```suggestion
 *		"<source_device>:<mount_target>:<fs_driver>:<options>",
```
I think there should be a comma here.

Reply:
@skuenzer AFAIK, the `uklibparam` design makes use of whitespaces in such cases :thinking:.

Reply:
Yes, no commas. This has actually a pragmatic reason because that missing comma saves us parsing code. We can just rely on `ukargparse` for doing the argument split and array parsing. `ukargparse` also handles quotes and escaped quotes correctly, so does uklibparam.